portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -27,6 +27,6 @@
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
     response.setContentType("text/html;");
-    response.getWriter().println("<h1>Hello world!</h1>");
-  }
+    response.getWriter().println("<h1>Hello Adrian!</h1>"); 
```

Comment:
Looks like this is not being used in the front end code. I believe you plan to add this later?